libcloud/test/container/test_rancher.py

```diff
@@ -157,6 +157,10 @@ def test_get_container(self):
         self.assertEqual(container.extra['environment'],
                          {'STORAGE_TYPE': 'file'})
 
+    def test_start_container(self):
+        container = self.driver.get_container("1i31")
+        container.start()
```

Comment:
You should really put assertions in these test methods, assert the returned instance of Container and the ID/name that it returns. same applies for the other methods. I should have spotted that last time.